### DISCUSSION TITLE
Include ibm.storage_virtualize collection

### DISCUSSION THREAD

**Discussion by Shilpi-J** [ansible-collections/ansible-inclusion#65] [Resolved reviews]
The collection ibm.spectrum_virtualize is already part of Ansible community. However, due to rebranding, the name spectrum has been changed to storage. We have created the new public GH repo for the same. The new collection is same as the old one with new feature enhancements that are done every quarter. 

Intent is to include ibm.storage_virtualize collections in Ansible community release. Please validate and provide your feedback for the same.
Ansible Galaxy: https://galaxy.ansible.com/ui/repo/published/ibm/storage_virtualize/
GitHub: https://github.com/ansible-collections/ibm.storage_virtualize
Issues Tracker: https://github.com/ansible-collections/ibm.storage_virtualize/issues
GitHub Handles:
Is the collection part of Automation Hub: Yes
We meet [Collection Requirements](https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst): Yes

**Comment by felixfontein** [comment]
@Shilpi-J please take a look at https://github.com/ansible-community/ansible-build-data/#renaming-a-collection

**Comment by Shilpi-J** [comment]
@felixfontein Yes, we have followed this link only to rename the collection.
The new collection ibm.storage_virtualize version 2.0.0 is backward compatible to ibm.spectrum_vritualize and we have marked the old collection deprecated, updated its README to redirect to the new collection. Also, the old collection is now archived.

Now, we want the new collection to be part of community

**Reply by Andersson007** [reply]
@felixfontein thanks for the link!

Let's continue the process in https://github.com/ansible-community/ansible-build-data/issues/306

**Comment by Andersson007** [comment]
Not to confuse folks, as the work is happening in https://github.com/ansible-community/ansible-build-data/issues/306, i'll move this discussion to anaother category. FYI
